ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -203,33 +216,38 @@ Item {
                 id: chatRepeater
                 model: parentModule && parentModule.model
 
-                ChatContentView {
+                Loader {
                     width: parent.width
                     height: parent.height
-                    visible: !root.rootStore.openCreateChat && model.active
-                    chatId: model.itemId
-                    chatType: model.type
-                    chatMessagesLoader.active: model.loaderActive
-                    rootStore: root.rootStore
-                    contactsStore: root.contactsStore
-                    emojiPopup: root.emojiPopup
-                    stickersPopup: root.stickersPopup
-                    stickersLoaded: root.stickersLoaded
-                    isBlocked: model.blocked
-                    onOpenStickerPackPopup: {
-                        root.openStickerPackPopup(stickerPackId)
-                    }
-                    onShowReplyArea: (messageId) => {
-                                         d.showReplyArea(messageId)
-                                     }
-                    onForceInputFocus: {
-                        chatInput.forceInputActiveFocus()
-                    }
+                    active: model.type !== Constants.chatType.category && model.type !== Constants.chatType.unknown
```

Comment:
I added this Loader, because it's completely useless to create a `ChatContentView` for a category. And for unknown types too obviously